kncaputo/rancid-tomatillos
# Feature/router

## What's this PR do?
- This PR adds React Router package to the project
- User can navigate the pages of our app via URL paths 
- Conditional rendering has been replace with Route Components for MovieContainer and MovieDetails

## Where should the reviewer start?
- App/App.js: line 67
- MovieCard/MovieCard.js: line 7

## How should this be manually tested?
- Navigate from home page(`http://localhost:3000/movies`) to MovieDetails by clicking on a movieCard 
- Should see movie details with URL `http://localhost:3000/movies/(movieId#)`

## Questions:
- How do we redirect to home page using router if error occurs
